src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -53,47 +52,46 @@
   static final int WEB_ID_ERROR = 2;
   static final int GENERIC_ERROR = 254;
 
-  static final String AUTH_ERROR_STRING = "Authorization has been denied for this request.";
   static final String WEB_ID_ERROR_STRING = "Unknown or invalid WebID format:";
 
-  // to be used in switch statements on which comms type to use
-  static final String communicationTypePre2019 = "piwebapi";
-  static final String communicationTypePost2019 = "omf";
-
-  // Error message to be displayed for internal comminication errors in this file
-  private static final String comsErrMsg =
-      "Internal Error OSIsoftServer.java: communication type is not valid";
-
-  private static StringBuilderLite batchBuffer;
-  private static final int STRING_BUILDER_NUM_CHARS = 500000;
-  private static int batchCount;
 
   private static SimpleDateFormat dateFormat;
 
-  private String omfTypeJson;
-
-  public OSIsoftServer(String ip, String login, String webID, String name) {
-    serverIP = ip;
-    authCredentials = login;
-    dbWebID = webID;
-    flexyName = name;
-    targetURL = "https://" + serverIP + "/piwebapi/";
+  /**
+   * Constructs the OSIsoftServer object.
+   *
+   * @param deviceName the name of the device
+   */
+  public OSIsoftServer(String deviceName) {
+    flexyName = deviceName;
+    targetURL = "https://" + OSIsoftConfig.getServerIP() + "/piwebapi/";

```

Comment:
Why are you saving this everywhere and not just accessing it from the config?

Reply:
yeah I had missed changing this when you mentioned it on the other PR. I moved a couple of things from the OSIsoft server to the config class when I changed this